### DISCUSSION TITLE
IVAN use of new columns

### DISCUSSION THREAD

**Discussion by richard-sisson** [australiangeomechanics/agsformat#63] [Q&A] [answered]
Hi,

Can you please clarify the use of these new headings in the IVAN group:
IVAN_VTID - Torque device ID  -> we assume this is the ID of the torque gauge device, referenced on calibration sheets
IVAN_VNID - Vane ID -> we assume this is the vane blade ID, referenced on calibration sheets (i.e. 19mm, A)?

Thanks
Richard

**Comment by SolidRockProgrammer** [comment, marked as answer]
I think the examples and descriptions are clear, they are both some type of identifier/code for the hardware
